### PR TITLE
EZP-29410: As an Editor I want to be informed that I can't edit CI because of missing policies.

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | [https://jira.ez.no/browse/EZP-29410](https://jira.ez.no/browse/EZP-29410)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

To perform a conflict check action content/versionread permission is needed. If the user has no policy, he will be informed in the notification.

screen shot 2019-02-07 at 3 59 49 pm

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/button.content.edit.js [hunk 1]
```diff
@@ -34,6 +35,10 @@
             $('#version-draft-conflict-modal').modal('show');
         };
 
+        const showError = (error) => {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/button.content.edit.js]: No empty line between `const`

Comment (sunpietro) [reply src/bundle/Resources/public/js/scripts/button.content.edit.js]: Please make it a one liner: `const showError = (error) => showErrorNotification(error);`

Comment (dew326) [reply src/bundle/Resources/public/js/scripts/button.content.edit.js]: I have a better idea. Why not call `showErrorNotification` directly in `then()`?

Comment (mikadamczyk) [reply src/bundle/Resources/public/js/scripts/button.content.edit.js]: @dew326 @sunpietro is it ok now?

File: src/bundle/Resources/public/js/scripts/button.content.edit.js [hunk 2]
```diff
@@ -43,11 +44,13 @@
             // Otherwise we can go to Content Item edit page.
             if (response.status === 409) {
                 response.text().then(showModal);
+            } else if (response.status === 403) {
+                response.text().then((error) => showErrorNotification(error));
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/button.content.edit.js]:
```suggestion
                response.text().then(showErrorNotification);
```

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by mikadamczyk** [comment]
@lserwatka done

**Pull request merged**